app/views/components/gds/removeButton.scala.html

```diff
@@ -23,7 +23,7 @@
 @(value: Option[String] = None, hiddenLabel: Option[String] = None)(implicit messages: Messages)
 
 @govukButton(Button(
- classes = "govuk-button--secondary",
+ classes = "govuk-button--secondary govuk-!-margin-0",
```

Comment:
Nit: Just making sure: this change is to be implemented for all remove buttons in the service?

Reply:
I don't know, probably yes.

Reply:
Should we go with with buttons or links?